src/tests/ftest/util/server_utils_params.py

```diff
@@ -443,6 +443,9 @@ class EngineYamlParameters(YamlParameters):
             "DAOS_POOL_RF=4",
             "CRT_EVENT_DELAY=1",
             "DAOS_VOS_AGG_GAP=25",
+            # FIXME disable space cache since some tests need to verify instant pool space
+            # changing, this global setting to individual test setting once in follow-on PR.
+            "DAOS_POOL_SPACE_CACHE_INTVL=0",
```

Comment:
Seems like we should be implementing this the other way around and adding it only to the tests that need this set in each of their test yamls. @daltonbohning thoughts?

Reply:
I think this is fine, setting it to zero makes all tests behave exactly same as today (no cache for gathering space information, because some tests need to verify space change immediately, which is not required in real world),  but it has scalability issue on Aurora if millions of processes try to get space information at the same time, so the default behavior is enabling cache even it's disabled for CI.